samanthas_telegram_bot/conversation/user_data.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+from typing import Literal
+
+from samanthas_telegram_bot.conversation.constants_enums import Role
+
+
+@dataclass
+class TeacherPeerHelp:
+    """A class that comprises boolean flags for experienced teachers' willingness to help their
+    peers.
+    """
+
+    can_consult_other_teachers: bool = None
+    can_help_with_children_group: bool = None
+    can_help_with_materials: bool = None
+    can_check_syllabus: bool = None
+    can_give_feedback: bool = None
+    can_invite_to_class: bool = None
+    can_work_in_tandem: bool = None
+
+
+@dataclass
+class UserData:
+    # TODO sets instead of lists?
+    locale: str = None
+    first_name: str = None
+    last_name: str = None
+    source: str = None
+    tg_username: str = None
+    phone_number: str = None
+    email: str = None
+    role: Role = None
+    # no datetime objects are needed to achieve results needed for the bot
+    utc_offset_hour: int = None
+    utc_offset_minute: int = None
+    time_slots_for_day: dict = None
```

Comment:
I strongly advise don't use raw dict as a typing. It should be dataclass or TypedDict. Because we need to understand internal scheme of this dict

Reply:
I'll add it as a TODO. In this PR I wanted to focus on the conversation, I know that `UserData` will need reworking, given changes in models since I started writing the bot.

Reply:
Ok! :)